swan-cern/templates/fluentd/fluentd_outputs.conf.yaml

```diff
@@ -0,0 +1,27 @@
+{{ if .Values.fluentd.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluentd-outputs-conf
+  namespace: {{ .Release.namespace }}
+data:
+  outputs.conf: |-
+      # Push logs to MONIT endpoint
+      <match {logs,metrics}>
+          @type http
+          endpoint_url    {{ .Values.fluentd.output.endpoint }}
+          serializer      json
+          http_method     post
+          ssl_no_verify   false
```

Comment:
This is already false by default as per:

https://github.com/fluent-plugins-nursery/fluent-plugin-out-http